nofos/composer/templates/composer/subsection_create.html

```diff
@@ -0,0 +1,245 @@
+
+  <form class="form" method="post">
+    {% csrf_token %}
+    <fieldset class="usa-fieldset">
+
+      {% if form.non_field_errors %}
+        <legend class="legend--bootstrap">
+          <div class="usa-error-message">
+            Error: {{ form.non_field_errors.0 }}
+          </div>
+        </legend>
+      {% endif %}
+
+      <!-- name -->
+      <div class="form-group">
+        {% with id="name" label=form.name.label value=form.name.value hint=form.name.help_text error=form.name.errors.0 input_type=form.name|input_type %}
+          {% include "includes/_text_input.html" with id=id label=label value=value error=error input_type=input_type safe_hint=True only %}
+        {% endwith %}
+
+      <!-- tag -->
+      <div class="form-group width-mobile">
+        <label class="usa-label" for="tag">Heading level</label>
+        <div class="usa-hint" id="tag--hint-1">
+          Required if name exists. {% if prev_subsection_with_tag %}Last heading: <span class="previous-subsection--tag">{{ prev_subsection_with_tag.tag }}</span>.{% endif %}
+        </div>
+        {% if form.errors.tag %}
+          <div class="usa-error-message" id="tag--error">
+            {{ form.errors.tag.0 }}
+          </div>
+        {% endif %}
+        <select {% whitespaceless %}
+          class="
+                usa-select
+                border-2px
+                  {% if form.errors.tag %}
+                    border-secondary-dark
+                  {% endif %}
+              "
+        {% endwhitespaceless %} class="usa-select border-2px" name="tag" id="tag" aria-describedby="tag--hint-1">
+          <option value="">---------</option>
+          <option value="h3" {% if form.tag.value == 'h3' %}selected{% endif %}>Heading 3</option>
+          <option value="h4" {% if form.tag.value == 'h4' %}selected{% endif %}>Heading 4</option>
+          <option value="h5" {% if form.tag.value == 'h5' %}selected{% endif %}>Heading 5</option>
+          <option value="h6" {% if form.tag.value == 'h6' %}selected{% endif %}>Heading 6</option>
+          <option value="h7" {% if form.tag.value == 'h7' %}selected{% endif %}>Heading 7</option>
+        </select>
+      </div>
+
+      <!-- callout_box -->
+      <div class="form-group margin-top-4 width-mobile">
+        <div class="usa-checkbox">
+          <input class="usa-checkbox__input usa-checkbox__input--tile" id="callout_box" type="checkbox" name="callout_box">
+          <label class="usa-checkbox__label" for="callout_box">Is callout box?
+            <span class="usa-checkbox__label-description">Callout boxes <span aria-hidden="true">(📦)</span> use an accent color to call attention to important content.</span>
+          </label>
+        </div>
+      </div>
+
+      <!-- Add instructions -->
+      <details class="form-group margin-top-4">
+        <summary><span class="text-bold">Add instructions ?</span></summary>
+        <div class="form-group margin-top-4">
+          <label
```

Comment:
Let's throw in some `usa-prose` here that copies the text we have on the "Edit instructions" page:

> - Edit instructions that OpsDivs and writers will review.
> - This content will automatically be locked and writers will not be able to make changes.